python/cudf/cudf/core/single_column_frame.py

```diff
@@ -359,3 +363,23 @@ def nunique(self, dropna: bool = True):
         if self._column.null_count == len(self):
             return 0
         return self._column.distinct_count(dropna=dropna)
+
+    def _get_from_column(self, arg) -> Union[ScalarLike, ColumnBase]:
```

Comment:
Get from column isn't super descriptive name, how about `_get_elements()`? -- or maybe that's equally bad.